makefile

```diff
@@ -38,8 +38,7 @@ dist-gen: clean composer
 	@cp -r composer.lock                    Lychee
 	@cp -r index.php                        Lychee
 	@cp -r LICENSE                          Lychee
-	@cp -r readme.md                        Lychee
-	@cp -r server.php                       Lychee
```

Comment:
I am not sure about that line.

Reply:
Why not? What do mean by "your are not sure"?

Reply:
Why, I thought it was done on purpose. Why do we want to include `server.php` into the distribution? As far as I understand it, the file is only useful during development if you run a minimalistic webserver from within Artisan. That is also the reason why I asked what @ildyria meant by "he is not sure".

Reply:
Just delete it (but also the actual server.php file if it is not in the Laravel 9 repo).

Reply:
also works for me.